glossary/SSL.md

```diff
@@ -0,0 +1,5 @@
+### SSL
+
+Secure Sockets Layer known as SSL is a protocol for transferring private data across the Internet.
```

Comment:
> Secure Socket Layer, commonly known as SSL or TLS, is a set of protocols and standards for...

Even though TLS is actually Transport Layer Security, the two terms are interchangeable since they are applied at the exact same layer in the exact same way(and if you're actually using SSL3 or below these days... I'd be surprised)